README.md

```diff
@@ -1,35 +1,43 @@
-[![Build status](https://ci.appveyor.com/api/projects/status/hu7qlgxrh2j0i35y/branch/master?svg=true)](https://ci.appveyor.com/project/ILMTitan/google-cloud-tfs/branch/master)
-[![codecov](https://codecov.io/gh/GoogleCloudPlatform/google-cloud-tfs/branch/master/graph/badge.svg)](https://codecov.io/gh/GoogleCloudPlatform/google-cloud-tfs)
+[![Build status][AppVeyorBadge]][AppVeyorLink]
+[![codecov][CodeCovBadge]][CodeCovLink]
 
-# ![GCPLogo][GCPLogo] Cloud Tools for Team Foundation Server
+# ![GcpLogo][GcpLogo] Cloud Tools for Team Foundation Server
 
 This repository contains an extension for Team Foundation server and Visual
 Studio Team Services. The extension provides a new service endpoint and several
 build tasks for interacting with Google Cloud Platform.
 
-## Documentation
-
-For Documentation on using the extension, see [DETAILS.md](DETAILS.md)
-
 ## Installation
 
 The latest stable version can be found on the
 [Visual Studio Marketplace][Marketplace] or in the [GitHub Releases][GitHubReleases]
 
-The latest development version is available from [AppVeyor][Appveyor].
+The latest development version is available from [AppVeyor][AppVeyor].
 
 You can [build the vsix package from source](#Build), and then upload and
 install the extension from the extension management page of TFS.
 
+## Documentation
+
+For Documentation on using the extension, see [DETAILS.md](DETAILS.md)
+
+## Support
+
+Issues are tracked on [GitHub][GitHubIssues].
+Questions can be asked on StackOverflow
+  - [Ask a question][StackOverflowAsk]
+  - [Browse asked questions][StackOverflowBrowse]
+
 ## Build
 
 ### Prerequisites
 
-  - [PowerShell][PowerShell]. Installed by default on modern windows platforms.
-  - [Node.js][Node]. Download and install from the website.
-  - [npm][npm]. Should be installed with the Node.js install.
-  - [TypeScript][TypeScript]. Install with `npm install -g typescript`
-  - [tfx][tfs-cli] The tfs cli. Install with `npm install -g tfx-cli`
+  - [PowerShell][PowerShell]: Installed by default on modern windows platforms.
+  - [Node.js][Node]: Download and install from the website.
```

Comment:
Actually one more thing.  This should probably mention 6.x must be used

Reply:
I will fix this dependency issue in a separate PR.